uikit/src/main/java/com/midtrans/sdk/uikit/activities/PaymentMethodsActivity.java

```diff
@@ -332,6 +332,7 @@ public void onFailure(Token token, String reason) {
 
                 @Override
                 public void onError(Throwable error) {
+                    enableButtonBack(true);
```

Comment:
why is this change here?

Reply:
this is needed to fix the issue can't press the back button on the error screen, I just fix some issue on this branch also